transaction/transaction_test.go

```diff
@@ -0,0 +1,12 @@
+package transaction
+
+import "testing"
+
+func TestSettingFailToString(t *testing.T) {
+	trans := Transaction{}
+	trans.Fail = "Fail"
```

Comment:
I'm not sure this test is accomplishing much. Is this more to ensure nobody changes the field in the future (i.e. the test is that the test compiles)?